install.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env zsh
+
+conda activate base
+yes | conda create --name fpst_venv
+yes | conda activate fpst_venv
+yes | conda install -c conda-forge fsleyes
+yes | conda install -c conda-forge wxpython=4.0.7
+cd ..
+python3 -m pip install ./fsleyes-plugin-shimming-toolbox
+python3 -m pip install --user pipx
+curl -L https://github.com/shimming-toolbox/shimming-toolbox/archive/refs/tags/v0.1-beta.tar.gz > shimming-toolbox-0.1-beta.tar.gz
+gunzip -c shimming-toolbox-0.1-beta.tar.gz | tar xopf -
+cd shimming-toolbox-0.1-beta
+pipx install . --force
```

Comment:
Tried it out but `pipx` needs to be on the `PATH`, `pipx ensurepath` does this but requires to reload the terminal.
![Screen Shot 2021-03-24 at 22 07 36](https://user-images.githubusercontent.com/47249340/112408151-a945e500-8ced-11eb-86f3-f17403bba646.png)

Reply:
Oh interesting! I must have missed that because I had already installed `pipx`. I'll take a look.

Reply:
This *should* fix it: https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/2/commits/03ebf252c868b5d86d1d52a4c67a3998add1bad5